library/Freckle/App/Kafka/Consumer.hs

```diff
@@ -164,31 +164,39 @@ runConsumer
      , HasCallStack
      )
   => Timeout
+  -> Int
   -> (a -> m ())
```

Comment:
Is this the right shape for the handler? Should it be `[a] -> m ()` instead? 

Seems like it'd more flexible for the client to do "smart" things if we hand them the entire batch. I'm thinking things like doing concurrent things or grouping stuff together.

Reply:
Good question. I defer to you.

Practically speaking, I'd put my money on consumers _not_ doing smart things, but it's also trivial for them to do their own `traverse_`.

Reply:
I think `[a] -> m ()` also implies that we can't store or commit offsets on each event, so I guess for the first attempt at this I can go with the `a -> m ()` shape and see what happens.